src/release/breaking-changes/paint-enableDithering.md

```diff
@@ -0,0 +1,130 @@
+
+In addition, the dithering documentation states support is _only_ for gradients.
+
+## Background
+
+[`Paint.enableDithering`][] was added as a global option in [PR 13868][] as
+a response to [Issue 44134][], which reported that gradients in Flutter had
+visible banding artifacts:
+
+> Gradients currently have a lot of color banding on all devices, and it looks
+> very weird when using the pulse animation too. A solution is to make the
+> gradients opaque, and to use dithered gradients with Skia. Dithered gradients
+> aren't currently exposed, so adding a dither parameter to dart:ui's Paint
+> class would be nice. We'd be able to manually draw our gradients with a
+> CustomPainter.
+
+![Example of banding](https://user-images.githubusercontent.com/30870216/210907719-4f4a1a8d-e28a-4d39-9e99-3635a26a0c74.png)
+
+In [Issue 118073][], it was reported that gradients in our new [Impeller][]
+backend that banding artifacts were visible in some gradients, and later
+discovered the (rarely used) [`Paint.enableDithering`][] setting was not
+supported in Impeller.
+
+After adding dithering support to Impeller ([PR 44181][], [PR 44331][],
+[PR 44522][]), and reviewing the perfomance impact of dithering (negligble),
+and:
+
+1. Consensus that gradients look good by default: [Issue 112498][].
+1. Having a global option was intended to be deprecated: [PR 13868][].
+
+... it was decided that Flutter would:
+
+1. Make dithering enabled by default.
+1. Deprecate the global option.
+1. Remove the global option in a future release.
+
+As part of that process, the ability for dithering to effect anything other than
+gradients was removed in [PR 44730][], [PR 44912][]. That was done to ease the
+process of migrating, because Impeller will never support dithering for anything
+but gradients.
+
+## Migration guide
+
+We expect that most users and libraries will not need to make any changes.
+
+For users that maintain golden tests, you may need to update your golden images
+to reflect the new default. For example, if you use [`matchesGoldenFile`][]
+to test a widget that contains a gradient:
+
+```shell
+flutter test --update-goldens
+```
+
+We don't expect this to be a common case, but if you do need to disable
+dithering temporarily, you can do so by setting the `enableDithering` property
+in your `main()` (either in an app or test):
+
+```diff
+void main() {
++ // TODO: Remove this after XYZ.
++ Paint.enableDithering = false;
+
+  runApp(MyApp());
+}
+```
+
+As the plan is to _permanently_ remove the `enableDithering` property, please
+provide feedback in [Issue 112498][] if you have a use case that requires
+disabling dithering (due to performance, crashes).
+
+If for some reason you _must_ draw gradients without dithering, you'll need to
+write your own custom shader. Describing that is out of the scope of this
+migration guide, but you can find some resources and examples:
+
+- [Writing and using fragment shaders][]
+- [`linear_gradient_ssbo_fill.frag`]
```

Comment:
This is sort of a bad example because it uses features that aren't exposed in the regular fragment program API.  A better example might be something like:

https://github.com/jonahwilliams/awesome_gradients/blob/main/shaders/hsl_linear_gradient.frag

Reply:
Oh ok, great, thanks for pointing that out! Fixed.